server/models.py

```diff
@@ -78,6 +81,15 @@ def __unicode__(self):
   def __str__(self):
     return self.__unicode__()
 
+class VendItem(models.Model):
+  name = models.TextField(default = '')
+  price = models.PositiveIntegerField(default = 0) # Price in pence
```

Comment:
As I said on the commit itself, you could do something like lhsid() here to make the admin interface look a little nicer